pkgs/build-support/node/build-npm-package/hooks/get-deps.py

```diff
@@ -0,0 +1,14 @@
+from glob import glob
```

Comment:
> I think it would be a lot more consistent to do this in bash with `jq`, we really dont need to depend on python for this.

I found this to be much saner, especially because of the differing types for one key name -- I don't even want to think how you'd do that with `jq`.

Reply:
And here's this script implemented as a `jq` filter, for reference:
```
if .packages != null then .packages | keys | map(select(. != "")) | join("\n") else .dependencies | keys | map("node_modules/" + .) end
```